extensions/amp-story/1.0/_locales/es.js

```diff
@@ -73,19 +73,19 @@ export default {
     string: 'Tumblr',
   },
   [LocalizedStringId.AMP_STORY_SHARING_PROVIDER_NAME_TWITTER]: {
-    string: 'Gorjeo',
+    string: 'Twitter',
   },
   [LocalizedStringId.AMP_STORY_SHARING_PROVIDER_NAME_WHATSAPP]: {
-    string: 'Whatsapp',
+    string: 'WhatsApp',
   },
   [LocalizedStringId.AMP_STORY_SYSTEM_LAYER_SHARE_WIDGET_LABEL]: {
     string: 'Compartir',
   },
   [LocalizedStringId.AMP_STORY_WARNING_DESKTOP_SIZE_TEXT]: {
-    string: 'Ampliar su ventana para ver esta experiencia',
+    string: 'Ampliar la ventana para ver esta experiencia',
   },
   [LocalizedStringId.AMP_STORY_WARNING_EXPERIMENT_DISABLED_TEXT]: {
-    string: 'Debe habilitar el experimento amp pisos para ver este contenido',
+    string: 'Debe habilitar el experimento amp-stories para ver este contenido',
```

Comment:
nit: `amp-story` but we we'll probably want to delete this anyways since it's not used anymore.

/cc @newmuis

Reply:
Good catch, corrected it here, was right in `es-419`. See my comment in the other one, I agree deleting both is best long term.

Reply:
We can deprecate it ourselves later, as it's not the purpose of this PR, thanks for the fix!